csrc/host_ir/executor.cpp

```diff
@@ -149,6 +149,22 @@ void HostIrExecutor::handle(Wait* wait) {
   works_.erase(communication);
 }
 
+void HostIrExecutor::handle(kir::ForLoop* for_loop) {
```

Comment:
@naoyam wdyt about the namespace? As said in the PR description, `kir` isn't appropriate any more since we'll also use it in host IR.

Reply:
How about just moving it to the top-level namespace? It's in the `kir` namespace because it's only valid in the Kernel IR, but now that if it's used also in the Host IR, placing it inside the `kir` namespace doesn't make sense. Perhaps, we should have a namespace like `nvfuser::ir`, but that's a separate discussion.

Reply:
Ok, is it ok if I do that in a separate PR once this one is merged?

Reply:
done in https://github.com/NVIDIA/Fuser/pull/2453